docs/definition_of_done.md

```diff
@@ -0,0 +1,20 @@
+---
+id: definition_of_done
+title: Definition of Done
+---
+The Cloud Engineering Team's Definition of Done is broken down into the following:
+
+## Approved by the end user
+Work that has complete needs to be approved by the end user to ensure that the requirements have been fully met.
+
+## Passes automated testing
+The code passes automated tests and will not be considered done until all tests pass.
+
+ ## Documentation is updated and centrally located
```

Comment:
Does the markdown need to be reverse indented here...*goes off to find out if there's a word for that*

Reply:
unindented? haha but yes i noticed it just after i PRed